dataset/sources/dataverse.py

```diff
@@ -0,0 +1,163 @@
+from core.utils.utils import fetch_data
+from dataset.models import Affiliation, Dataset, Dataverse, File, Publication, Author, Publisher
+from thematic_areas.models import ThematicArea
+from vocabulary.models import Keyword
+
```

Comment:
É possível esse módulo conter somente o **load_from_data_scielo** e as outras funções estarem disponíveis nas classes para reuso não somente aqui, mas para quem for gerar possível novos código manipulando esses dados.